.eslintrc.js

```diff
@@ -65,4 +65,11 @@ module.exports = {
     'require-await': 'error',
     'no-return-await': 'error',
   },
+  settings: {
+    'import/resolver': {
+      node: {
+        paths: ['src'],
+      },
+    },
+  },
```

Comment:
I had to apply https://stackoverflow.com/a/42498220 even though I'm not confident what it really does

Reply:
Not entirely sure either, but I know I also fought with ESLint import errors previously in related work so glad there is something that appears to have worked.